README.md

```diff
@@ -55,3 +55,68 @@ See `k8s-jenkins-cac.tf` in your project.
 There are 2 environments involved:
 * *target*: this is where the services will be deployed (where the version changes happen).
 * *upstream*: this is where the system tests are run. It is also the source for the *versions* of the apps.
+
+---
+
+#### Environment Config
+
+The config for an environment is stored in a json file called `config.json`
```

Comment:
please clarify the exact path of the file, e.g. "an env named 'dev' should be in a file 'dev/config.json'

Reply:
Fixed